pyproject.toml

```diff
@@ -36,7 +36,29 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry.dev-dependencies]
 toml = ">=0.10.2"
 configobj = "^5.0.8" # moved to dev dependencie so security alert does not propagate further up
+exasol-toolbox = ">=0.14.0"
 
 
 [tool.poetry.scripts]
 exaslct = 'exasol_script_languages_container_tool.main:main'
+
+[[tool.mypy.overrides]]
+module = ["exasol_integration_test_docker_environment.*", "luigi.*", "docker.*",
```

Comment:
While I see this, we probably add a py.typed file

https://blog.whtsky.me/tech/2021/dont-forget-py.typed-for-your-typed-python-package/

Reply:
Done